src/modules/governance/proposal/VotersListContainer.tsx

```diff
@@ -0,0 +1,189 @@
+  const queryParams = `?proposal=${proposalId}`;
+
+  const getVoterInfo = async () => {
+    if (error) setError(false);
+
+    try {
+      // Get proposal voters data
+      const resp = await fetch(votersUrl + queryParams);
+
+      if (resp.ok) {
+        const [yaes, nays]: GovernanceProposalTopVotersResponse = await resp.json();
+        // Transform data for UI, sort by highest voting power
+        const yesVoters: GovernanceVoter[] = yaes.map((v: GovernanceVoter) => {
+          const proposalVote = v.votingHistory.find((h) => h.proposal.id === proposalId.toString());
+          return {
+            ...v,
+            vote: 1,
+            proposalVotingPower: proposalVote?.votingPower ?? 0,
+          };
+        });
+        const noVoters: GovernanceVoter[] = nays.map((v: GovernanceVoter) => {
+          const proposalVote = v.votingHistory.find((h) => h.proposal.id === proposalId.toString());
+          return {
+            ...v,
+            vote: 0,
+            proposalVotingPower: proposalVote?.votingPower ?? 0,
+          };
+        });
+        const votersData: VotersData = {
+          yaes: yesVoters.sort(sortByVotingPower),
+          nays: noVoters.sort(sortByVotingPower),
+          combined: yesVoters.concat(noVoters).sort(sortByVotingPower),
+        };
+        setVoters(votersData);
+      } else {
+        setError(true);
+      }
+    } catch (e: unknown) {
+      console.error(e);
+      setError(true);
+    }
+    setLoading(false);
+  };
+
+  useEffect(() => {
+    getVoterInfo();
+  }, [forVotes, againstVotes]); /* eslint-disable-line react-hooks/exhaustive-deps */
+
+  const handleOpenAllVotes = () => {
+    setVotersModalOpen(true);
+  };
+
+  const listHeaderComponent = (
+    <Row sx={{ mb: 3 }}>
+      <Typography variant="subheader2" color="text.secondary">
+        <Trans>Top 10 addresses</Trans>
+      </Typography>
+      <Typography variant="subheader2" color="text.secondary">
+        <Trans>Votes</Trans>
+      </Typography>
+    </Row>
+  );
+
+  if (loading)
+    return (
+      <Box sx={{ mt: 8, mb: 12 }}>
+        {listHeaderComponent}
+        <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+          <CircularProgress size={24} sx={{ my: 4 }} />
+        </Box>
+      </Box>
+    );
+
+  if (error)
+    return (
+      <Box sx={{ mt: 8, mb: 12 }}>
+        {listHeaderComponent}
+        <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center', mt: 8 }}>
+          <Typography variant="helperText" color="error.main">
+            <Trans>Failed to load proposal voters. Please refresh the page.</Trans>
+          </Typography>
+        </Box>
+      </Box>
+    );
+
+  if (!voters || voters.combined.length === 0) return <Box sx={{ mt: 8 }} />;
+
+  return (
+    <Box sx={{ my: 8 }}>
+      {listHeaderComponent}
+      <VotersList compact={mdScreen} voters={voters.combined.slice(0, 10)} sx={{ my: 4 }} />
```

Comment:
Add `pr: 2` here and it will fix the scrollbar overlap

Reply:
But at 960px it's too much.

Reply:
It's really the only fix we can do without adjusting everything else, so this is the way for now.